src/components/chatX/ChatXBody.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ChatXMessage from './ChatXMessage';
+
+
+const ChatXBody = ({ messageList }) => {
+    let list = [];
+    if (messageList !== []) {
```

Comment:
This will always return true because `[] !== []`. You could just use `Array.isArray(messageList)` as you don't need to know if this array is empty.